### PR TITLE
community[minor],docs[minor]: Add ChromeAI chat model

### PR DESCRIPTION
**Pull request by bracesproul**

### CODE REVIEW

File: libs/langchain-community/src/chat_models/tests/chrome_ai.html [hunk 1]
```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
```
Comment (jacoblee93) [inline libs/langchain-community/src/chat_models/tests/chrome_ai.html:1]: How do we run this?

File: libs/langchain-community/src/chat_models/chrome_ai.ts [hunk 2]
```diff
@@ -0,0 +1,181 @@
+import {
```
Comment (jacoblee93) [inline libs/langchain-community/src/chat_models/chrome_ai.ts:1]: Let's put under `experimental` for now

File: libs/langchain-community/src/chat_models/chrome_ai.ts [hunk 3]
```diff
@@ -0,0 +1,181 @@
...
+    _options: this["ParsedCallOptions"],
+    runManager?: CallbackManagerForLLMRun
+  ): AsyncGenerator<ChatGenerationChunk> {
+    if (!this.session) {
+      throw new Error("Session not found. Please call `.initialize()` first.");
+    }
+    const textPrompt = formatPrompt(messages);
+
+    const stream = this.session.promptStreaming(textPrompt);
+    const reader = stream.getReader();
```
Comment (jacoblee93) [inline libs/langchain-community/src/chat_models/chrome_ai.ts:143]: nit: I think these are iterable in the browser

**Comment by jacoblee93** [comment]
Sweet!

File: libs/langchain-community/src/chat_models/tests/chrome_ai.html (hunk 1 above)
Comment (bracesproul) [reply libs/langchain-community/src/chat_models/tests/chrome_ai.html]: added better setup instructions

**Pull request merged**